docs/content/guide/component.ngdoc

```diff
@@ -109,13 +109,16 @@ change will be reflected in the parent component. For components however, only t
 the data should modify it, to make it easy to reason about what data is changed, and when. For that reason,
 components should follow a few simple conventions:
 
-  - Inputs are realized with `@` and `=` bindings
+  - Inputs are realized with `<` bindings. The `<` symbol denotes {@link $compile#-scope- one-way bindings} which are
```

Comment:
Another advantage of < (or = for that matter) is that the type of object is preserved, while @ only supports strings. Static literals are also unwatched automatically, so there's no real performance benefit afaik.